tests/components/homeassistant/triggers/test_homeassistant.py

```diff
@@ -1,29 +1,39 @@
 """The tests for the Event automation."""
-from unittest.mock import AsyncMock, patch
+from unittest.mock import patch
+
+import pytest
 
 import homeassistant.components.automation as automation
-from homeassistant.core import CoreState
+from homeassistant.core import CoreState, HomeAssistant
+from homeassistant.helpers.typing import ConfigType
 from homeassistant.setup import async_setup_component
 
 from tests.common import async_mock_service
 
 
-async def test_if_fires_on_hass_start(hass):
+@pytest.mark.parametrize(
+    "hass_config",
```

Comment:
This results in a slight change => hass_config will now be validated against the schema